ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -88,7 +90,7 @@ void CleanupAttributes()
             {
                 String name = (string)ca.ConstructorArguments[0].Value;
                 int idx;
-                if ((idx = name.IndexOf(", PublicKey=", StringComparison.Ordinal)) != -1)
+                if ((idx = name.IndexOfRegex(new Regex(", ?PublicKey="))) != -1)
```

Comment:
And it is a choice to secure the space, but why not generalize the fix and be sure more than a space or a tabulation are just also. It is a case so particular.